dotcom-rendering/window.guardian.d.ts

```diff
@@ -68,6 +68,11 @@ declare global {
 			onConsentChange?: (fn: Callback) => void;
 			getConsentFor?: (fn: Callback) => void;
 		};
+		mockLiveUpdate: (data: {
+			numNewBlocks: number;
+			html: string;
+			mostRecentBlockId: string;
+		}) => void;
```

Comment:
If you import onSuccess from that file, you can keep everything in sync.

```suggestion
		mockLiveUpdate: typeof onSuccess
```

Reply:
Sorry, you’ll also need to import the method in this file!

Reply:
I created `LiveUpdateType` here instead. Like this we don't need to export the function, which feels like a win in terms of reducing complexity in the code